docs/20201222.md

```diff
@@ -0,0 +1,64 @@
+## Muon Gun Efficiency Benchmark
+
+Here I document the GPU and CPU version benchmark for low pt muon gun efficiencies with GPU and CPU.
+
+### Copy-paste command lines
+
+Log on to phi3
+
+    git clone git@github.com:sgnoohc/TrackLooper-1.git
+    cd TrackLooper-1/
+    git checkout 463ec57
```

Comment:
is this current?

I was expecting instructions based on the SegmentLinking/TrackLooper repo

Reply:
It's a bootstrap paradox.
I can't point to the main repo tag unless we finish this pull request.

Reply:
shouldn't it at worst point to this PR `sgnoohc:master`?
... btw, it's not a good idea to make a PR using `master` as a topic branch.